Lib/fontmake/instantiator.py

```diff
@@ -541,9 +557,11 @@ def collect_kerning_masters(
     groups = designspace.default.font.groups
 
     locations_and_masters = []
+
     for source in designspace.sources:
-        if source.layerName is not None:
-            continue  # No kerning in source layers.
+        if source.layerName is not None and source.layerName != DEFAULT_LAYER_NAME:
+            if source != designspace.default:
```

Comment:
same thing here as for the font info above; ignore layerName, just continue `if source is not designspace.default`

Reply:
OK fixed.